### PR TITLE
Add counting of students with progress to students.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Also removes unused fields.

### CODE REVIEW

File: pegasus/helpers/professional_development_workshop_helpers.rb [hunk 1]
```diff
@@ -61,37 +61,22 @@ def generate_professional_development_workshop_teachers_report
...
-      #   lifetime = 0
-      #   levels = 0
-      # end
+        count
+
+      students_with_progress_count = DASHBOARD_DB[:followers].
+        where(user_id: teacher_user_id).
+        join(:users, id: :student_user_id).
+        join(:user_scripts, user_id: id).
+        where(script_id: [1, 17, 18, 19, 23]).
```
Comment (aoby) [inline pegasus/helpers/professional_development_workshop_helpers.rb]: Where do these Ids come from? Should these be constants?

Comment (ashercodeorg) [inline pegasus/helpers/professional_development_workshop_helpers.rb]: Probably. These are the CSF script IDs.

Comment (ashercodeorg) [inline pegasus/helpers/professional_development_workshop_helpers.rb]: Done (in that there is now a query to retrieve the IDs).

**Comment by aoby** [comment]
LGTM

**Pull request merged**